dev/image-guidelines.md

```diff
@@ -0,0 +1,335 @@
+
+![](image-guidelines/excalidraw_custom_fonts.png)
+
+Unfortunately when the Excalidraw diagrams are exported to SVG the custom fonts are not included. But we can switch the fonts by doing a text search & replace on the SVG files. This is explained [here](#fonts-in-svg-images)
+
+For normal text use the **Medium** font size. It's thickness should correspond closely to the thickness of lines and arrows set to **Extra Bold**.
+
+## Colors
+
+For **screenshot annotations** we use HTML `#228be6` for lines and text.
+This is one of the colors in Excalidraw's built-in palette and it contrasts well with Velociraptor's light theme.
+It's also light enough to be visible against dark backgrounds.
+
+For pure **line diagrams** we use black for lines and text, with de-emphasized elements being grey.
+
+Additional colors can be used to emphasize particular drawing objects.
+
+The default Excalidraw color palette is well designed, so only use those colors.
+This makes it easier to ensure that colors used in future can be matched those used in the past.
+
+## Excalidraw object library
+
+This file is an Excalidraw library containing reusable shape objects for our drawings.
+
+[library.excalidrawlib](library.excalidrawlib)
+
+If you find or create objects that you think will be useful in future drawings then please add them to this library, save the changes and commit the updated version back to the repo.
+
+
+## Screenshots
+
+### Screen size
+
+While we may do our daily work on screens with high display resolutions,
+the documentation will likely be read on smaller displays or possibly even printed.
+It needs to be legible on typical screens without a magnifying glass or zooming in.
+
+Ideally we want screenshots in the documentation to have a comfortably low resolution.
+This can be achieved by either:
+
+- only showing the part of the screen that the user needs to see.
+- using a constrained screen size for full-screen screenshots.
+
+These are explained below.
+
+#### Partial screens
+
+Screenshots that show only part of the screen area are relatively straightforward.
+
+![](image-guidelines/screen_partial.png)
+
+A useful trick in Chrome Dev Tools is that you can right-click on a DOM element and capture an image of just that element.
+![](image-guidelines/node_screenshot.png)
+
+#### Full screens
+
+When full screenshots are needed we can improve their legibility by simulating a smaller screen size.
+The "dev tools" of modern browsers allows you to simulate arbitrary screen sizes, including many common phone screens.
+Velociraptor's GUI is not a fully responsive web application but it's layout remains usable at quite small (by modern standards) screen resolutions.
+
+For full screenshots use one of the following 2 sizes:
+
+- 1280x960 (preferred)
```

Comment:
This is very important - we do need to be consistent with the sizes

Reply:
Are you saying 1280x960 should be an absolute requirement? It should be fine in most cases but I'm sure there are cases where it's not, like this one from your detection engineering post for example:
![live_sigma_detection](https://github.com/user-attachments/assets/d78f1dd2-8a00-411e-a98d-d02006d1dd00)
Discretion seems necessary.